### PR TITLE
Added project setup guidelines with commands

### LINKED ISSUE #30: [Docs] Add project setup guidelines with commands in `contributing.md`
#### Current State
The `contributing.md` is missing project setup command instructions. 

#### Desired State
Add project setup guidelines with commands in `contributing.md`
Refer to the Makefile to get setup commands.

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by HM-8**
**Notes for Reviewers**

This PR fixes #30

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,13 +7,128 @@ Not sure where to start? First, see the [newcomers welcome guide](https://docs.g
...
+
+### Set up your Local Development Environment
+Follow the following instructions to start contributing.
+
+**1.** Fork this repository.
+
+**2.** Clone your forked copy of the project.
+
+```
+git clone https://github.com/<your-github-username>/play.git
```
Comment (Nikhil-Ladha) [inline CONTRIBUTING.md:42]: Update it to the `ssh` URL

Comment (HM-8) [reply CONTRIBUTING.md:42]: Oh okay I'll change it

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,13 +7,128 @@ Not sure where to start? First, see the [newcomers welcome guide](https://docs.g
...
+
+```
+git commit -s -m "<commit subject>"
+```
+
+**13.** While you are working on your branch, other developers may update the `master` branch with their branch. This action means your branch is now out of date with the `master` branch and missing content. So to fetch the new changes, follow along:
+
+```
+git checkout master
+git fetch origin master
```
[the remark this answers is not in the document]
Comment (HM-8) [reply CONTRIBUTING.md:113]: Oh didn't notice. Will change it right away.

**Review by Nikhil-Ladha — APPROVED** [review]
LGTM

**Pull request merged**